packages/mailgun/src/models/v1/emails/request/send-mime-email-request/send-mime-email-request.ts

```diff
@@ -23,91 +24,22 @@ export interface SendMimeEmailRequest {
 
 export const transformSendMimeEmailRequestIntoApiRequestBody = (sdkRequest: SendMimeEmailRequest): FormData => {
   const formData = new FormData();
-  if (sdkRequest['to'] !== undefined && sdkRequest['to'] !== null) {
+  if (sdkRequest['to'] != null) {
     formData.append('to', sdkRequest['to']);
   }
-  if ('message' in sdkRequest && sdkRequest['message'] !== undefined && sdkRequest['template'] !== null) {
-    formData.append('message', sdkRequest['message'], {
-      filename: 'MimeMessage',
-    });
+  if ('message' in sdkRequest && sdkRequest['message'] != null) {
```

Comment:
All `emai` and some of `mime-emai`l fields are validated simple with simplified  `if (sdkRequest['XXX'] != null) {`
But not here (nor `template`) field.
Is it by purpose ?

Reply:
The check of the property existence was indeed a remain from the past. I've deleted it and updated the property check with the loose inequality operator `!=` (checks for value equality with type coercion)